icons/socketio/socketio-original-wordmark.svg

```diff
@@ -0,0 +1 @@
#010101"/><path d="m19.97 22.721c2.12 0 4.23 0 6.34 0.03-4.62 3.73-9.09 7.64-13.79 11.27 2.46-3.78 4.99-7.52 7.45-11.3z" fill="#010101"/><path d="m51.864 28.397c0.86101 0.47834 2.1286 0.81317 3.5397 0.81317 3.0853 0 4.7355-1.5307 4.7355-3.6354-0.02392-1.722-1.0045-2.7983-3.1092-3.5636-1.4828-0.52617-2.0329-0.88492-2.0329-1.6263 0-0.71751 0.574-1.2198 1.6263-1.2198 1.0284 0 1.9612 0.38267 2.4634 0.66967l0.574-2.0808c-0.66967-0.35875-1.7938-0.69359-3.0853-0.69359-2.7026 0-4.4007 1.6024-4.4007 3.6593-0.02392 1.3633 0.93276 2.6069 3.2049 3.3962 1.435 0.50226 1.9134 0.90884 1.9134 1.6981 0 0.76534-0.574 1.2915-1.8655 1.2915-1.0523 0-2.3199-0.4305-2.9896-0.83709zm14.98-11.337c-3.5158 0-6.0271 2.3439-6.0271 6.1706 0 3.731 2.5352 6.0031 5.8357 6.0031 2.9657 0 5.9792-1.9134 5.9792-6.1945 0-3.5397-2.3199-5.9792-5.7879-5.9792zm-0.07175 2.1525c2.009 0 2.8222 2.0808 2.8222 3.8985 0 2.3439-1.1719 3.9941-2.8461 3.9941-1.7938 0-2.8939-1.6981-2.8939-3.9463 0-1.9373 0.83709-3.9463 2.9179-3.9463zm15.434 7.2229c-0.57401 0.23917-1.2915 0.45442-2.3199 0.45442-2.009 0-3.5636-1.3633-3.5636-3.755-0.02392-2.1286 1.3154-3.7789 3.5636-3.7789 1.0523 0 1.7459 0.23917 2.2243 0.45442l0.52617-2.2004c-0.66967-0.31092-1.7698-0.55009-2.8461-0.55009-4.0898 0-6.4815 2.7265-6.4815 6.2184 0 3.6114 2.3678 5.9314 6.0031 5.9314 1.4589 0 2.6787-0.31092 3.3005-0.59792zm4.5283-14.446h-2.9418v16.981h2.9418v-3.9702l1.0045-1.1719 3.3484 5.1421h3.6114l-4.9269-6.8881 4.305-4.7595h-3.5397l-2.8222 3.7549c-0.31092 0.40659-0.64576 0.90884-0.93276 1.3633h-0.04783zm17.946 11.982c0.0478-0.26309 0.0957-0.69359 0.0957-1.2437 0-2.5591-1.2437-5.6683-5.0225-5.6683-3.731 0-5.6922 3.0374-5.6922 6.2662 0 3.5636 2.2243 5.8836 6.0031 5.8836 1.6742 0 3.0614-0.31092 4.0659-0.71751l-0.43051-2.0329c-0.88492 0.33484-1.8655 0.52617-3.2288 0.52617-1.8655 0-3.5158-0.90884-3.5875-3.0135zm-7.7969-2.0569c0.11958-1.1958 0.88492-2.8461 2.7026-2.8461 1.9612 0 2.4395 1.7698 2.4156 2.8461zm9.7661-7.2229v2.6309h-1.6742v2.2004h1.6742v5.477c0 1.5307 0.287 2.583 0.90884 3.2527 0.55009 0.59792 1.4589 0.95668 2.5352 0.95668 0.93276 0 1.6981-0.11958 2.1286-0.287l-0.0478-2.2482c-0.26309 0.07175-0.64576 0.1435-1.148 0.1435-1.1241 0-1.5068-0.74142-1.5068-2.1525v-5.1421h2.7983v-2.2004h-2.7983v-3.3244zm8.2115 14.541c1.1002 0 1.8177-0.81317 1.8177-1.8894 0-1.1241-0.74142-1.8894-1.8177-1.8894-1.0523 0-1.8177 0.78926-1.8177 1.8894 0 1.0763 0.74142 1.8894 1.8177 1.8894zm6.1546-0.26309v-11.648h-2.9657v11.648zm-1.4828-16.503c-0.98059 0-1.6503 0.69359-1.6503 1.6024 0 0.88492 0.64576 1.5785 1.6264 1.5785 1.0284 0 1.6742-0.69359 1.6742-1.5785-0.0239-0.90884-0.64576-1.6024-1.6503-1.6024zm8.929 4.592c-3.5158 0-6.027 2.3439-6.027 6.1706 0 3.731 2.5352 6.0031 5.8357 6.0031 2.9657 0 5.9792-1.9134 5.9792-6.1945 0-3.5397-2.3199-5.9792-5.7879-5.9792zm-0.0718 2.1525c2.009 0 2.8222 2.0808 2.8222 3.8985 0 2.3439-1.1719 3.9941-2.8461 3.9941-1.7938 0-2.8939-1.6981-2.8939-3.9463 0-1.9373 0.83709-3.9463 2.9179-3.9463z" fill="#000" stroke-width=".99654"/></g></svg>
```

Comment:
Missing an endline (this could cause some problems with some IDEs and OSes).
```suggestion
<svg version="1.1" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><g transform="matrix(.95867 0 0 .95867 -.70175 43.325)" fill="none" fill-rule="evenodd"><circle cx="21.693" cy="21.566" r="19.211" fill-opacity="0" stroke="#010101" stroke-width="3.5"/><path d="m16.09 20.631c4.59-3.75 9.08-7.63 13.77-11.27-2.46 3.78-4.99 7.51-7.45 11.29-2.11 0.01-4.22 0.01-6.32-0.02z" fill="#010101"/><path d="m19.97 22.721c2.12 0 4.23 0 6.34 0.03-4.62 3.73-9.09 7.64-13.79 11.27 2.46-3.78 4.99-7.52 7.45-11.3z" fill="#010101"/><path d="m51.864 28.397c0.86101 0.47834 2.1286 0.81317 3.5397 0.81317 3.0853 0 4.7355-1.5307 4.7355-3.6354-0.02392-1.722-1.0045-2.7983-3.1092-3.5636-1.4828-0.52617-2.0329-0.88492-2.0329-1.6263 0-0.71751 0.574-1.2198 1.6263-1.2198 1.0284 0 1.9612 0.38267 2.4634 0.66967l0.574-2.0808c-0.66967-0.35875-1.7938-0.69359-3.0853-0.69359-2.7026 0-4.4007 1.6024-4.4007 3.6593-0.02392 1.3633 0.93276 2.6069 3.2049 3.3962 1.435 0.50226 1.9134 0.90884 1.9134 1.6981 0 0.76534-0.574 1.2915-1.8655 1.2915-1.0523 0-2.3199-0.4305-2.9896-0.83709zm14.98-11.337c-3.5158 0-6.0271 2.3439-6.0271 6.1706 0 3.731 2.5352 6.0031 5.8357 6.0031 2.9657 0 5.9792-1.9134 5.9792-6.1945 0-3.5397-2.3199-5.9792-5.7879-5.9792zm-0.07175 2.1525c2.009 0 2.8222 2.0808 2.8222 3.8985 0 2.3439-1.1719 3.9941-2.8461 3.9941-1.7938 0-2.8939-1.6981-2.8939-3.9463 0-1.9373 0.83709-3.9463 2.9179-3.9463zm15.434 7.2229c-0.57401 0.23917-1.2915 0.45442-2.3199 0.45442-2.009 0-3.5636-1.3633-3.5636-3.755-0.02392-2.1286 1.3154-3.7789 3.5636-3.7789 1.0523 0 1.7459 0.23917 2.2243 0.45442l0.52617-2.2004c-0.66967-0.31092-1.7698-0.55009-2.8461-0.55009-4.0898 0-6.4815 2.7265-6.4815 6.2184 0 3.6114 2.3678 5.9314 6.0031 5.9314 1.4589 0 2.6787-0.31092 3.3005-0.59792zm4.5283-14.446h-2.9418v16.981h2.9418v-3.9702l1.0045-1.1719 3.3484 5.1421h3.6114l-4.9269-6.8881 4.305-4.7595h-3.5397l-2.8222 3.7549c-0.31092 0.40659-0.64576 0.90884-0.93276 1.3633h-0.04783zm17.946 11.982c0.0478-0.26309 0.0957-0.69359 0.0957-1.2437 0-2.5591-1.2437-5.6683-5.0225-5.6683-3.731 0-5.6922 3.0374-5.6922 6.2662 0 3.5636 2.2243 5.8836 6.0031 5.8836 1.6742 0 3.0614-0.31092 4.0659-0.71751l-0.43051-2.0329c-0.88492 0.33484-1.8655 0.52617-3.2288 0.52617-1.8655 0-3.5158-0.90884-3.5875-3.0135zm-7.7969-2.0569c0.11958-1.1958 0.88492-2.8461 2.7026-2.8461 1.9612 0 2.4395 1.7698 2.4156 2.8461zm9.7661-7.2229v2.6309h-1.6742v2.2004h1.6742v5.477c0 1.5307 0.287 2.583 0.90884 3.2527 0.55009 0.59792 1.4589 0.95668 2.5352 0.95668 0.93276 0 1.6981-0.11958 2.1286-0.287l-0.0478-2.2482c-0.26309 0.07175-0.64576 0.1435-1.148 0.1435-1.1241 0-1.5068-0.74142-1.5068-2.1525v-5.1421h2.7983v-2.2004h-2.7983v-3.3244zm8.2115 14.541c1.1002 0 1.8177-0.81317 1.8177-1.8894 0-1.1241-0.74142-1.8894-1.8177-1.8894-1.0523 0-1.8177 0.78926-1.8177 1.8894 0 1.0763 0.74142 1.8894 1.8177 1.8894zm6.1546-0.26309v-11.648h-2.9657v11.648zm-1.4828-16.503c-0.98059 0-1.6503 0.69359-1.6503 1.6024 0 0.88492 0.64576 1.5785 1.6264 1.5785 1.0284 0 1.6742-0.69359 1.6742-1.5785-0.0239-0.90884-0.64576-1.6024-1.6503-1.6024zm8.929 4.592c-3.5158 0-6.027 2.3439-6.027 6.1706 0 3.731 2.5352 6.0031 5.8357 6.0031 2.9657 0 5.9792-1.9134 5.9792-6.1945 0-3.5397-2.3199-5.9792-5.7879-5.9792zm-0.0718 2.1525c2.009 0 2.8222 2.0808 2.8222 3.8985 0 2.3439-1.1719 3.9941-2.8461 3.9941-1.7938 0-2.8939-1.6981-2.8939-3.9463 0-1.9373 0.83709-3.9463 2.9179-3.9463z" fill="#000" stroke-width=".99654"/></g></svg>

```